server/views/pages/search.njk

```diff
@@ -33,9 +33,12 @@
           <div class="grid__cell">
             <h3 class="{{ {s:'WB6'} | fontClasses }}">Suggested Topics</h3>
             {% componentV2 'tags', {id: 'search-landing', tags: [
+              {text: 'Quacks', url: '/search?query=quack+OR+quacks', bgColor: 'turquoise-text'},
+              {text: 'James Gillray', url: '/search?query=creator%3Ajames+gillray', bgColor: 'turquoise-text'},
+              {text: 'Botany', url: '/search?query=botany', bgColor: 'turquoise-text'},
+              {text: 'Optics', url: '/search?query=optics', bgColor: 'turquoise-text'},
+              {text: 'Sun', url: '/search?query=Sun', bgColor: 'turquoise-text'},
```

Comment:
Given that the search for `quack or quacks` and `quack OR quacks` gives the same number of results, I'm guessing insensitivity. Apols for letting my own typing quirks get in the way!